### PR TITLE
usage of javascript/node to allow cross-platform building

### PR DESCRIPTION
**Pull request by Osiris-Team**
At the current state the build fails when following the simple instructions in the README on Windows.
This fixes it.

## Summary by CodeRabbit

- **新功能**
	- 添加了用于清理开发工作区的脚本。
	- 添加了用于复制注入文件的脚本。
	- 添加了用于执行后安装步骤的脚本。
- **依赖更新**
	- 新增依赖：`"cross-env-shell": "^7.0.3"`。
- **脚本更新**
	- 更新了`postinstall`和`copy:inject`脚本为Node.js脚本。
	- 新增`clean`脚本用于清理工作区。

### CODE REVIEW

**Comment by sidmorizon** [comment]
Thanks for your contribution @Osiris-Team 

But we're not planning to introduce Python scripts into the project, you can consider converting the Python scripts to Node scripts.

**Comment by Osiris-Team** [comment]
Oh yeah that makes sense, didn't think about that!
-> Fixed

**Comment by huhuanming** [comment]
Thanks for your contribution @Osiris-Team.

but [development/scripts/copy_injected.js](https://github.com/OneKeyHQ/app-monorepo/pull/5825/files#diff-0a60283be399a546b76d2f424faca64f921ab600dc97a82f3b2a8be579104532) is still a python file.

Maybe you can make modifications based on this draft PR https://github.com/OneKeyHQ/app-monorepo/pull/5830

**Comment by Osiris-Team** [comment]
my bad forgot to copy over, fixed

File: package.json [hunk 1]
```diff
@@ -11,9 +11,10 @@
     "node": ">=20"
   },
   "scripts": {
-    "setup:env": "bash -c 'if [ ! -f .env ]; then cp .env.example .env; fi'",
```
Comment (huhuanming) [inline package.json:14]: Would it be better to write it like this?
```
"setup:env": "node -e \"const fs=require('fs');if(!fs.existsSync('.env'))fs.copyFileSync('.env.example','.env');\"",
```

**Comment by huhuanming** [comment]
and apps/desktop/dist/build_keytar.sh need to be fixed.

After all the files have been modified, I will help you package and test it.

**Review by sidmorizon — APPROVED** [review]

**Review by originalix — APPROVED** [review]

**Comment by Osiris-Team** [comment]
Is there something left to do?
Not sure because of all the bots commenting.

**Comment by huhuanming** [comment]
Thank you very much for your help. We have updated the script to JavaScript. Since the current content conflicts with the existing code, we have temporarily closed the PR. You are welcome to continue submitting relevant PRs.

**Pull request closed without merging**